spec/requests/compliment_page_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe 'Compliment me page', type: :request do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.